src/watertap_contrib/reflo/costing/watertap_reflo_costing_package.py

```diff
@@ -321,11 +321,7 @@ def build_integrated_costs(self):
                 expr=(
                     self.frac_heat_from_grid
                     == 1
-                    - (
-                        -1
-                        * energy_cost.aggregate_flow_heat
-                        / treat_cost.aggregate_flow_heat
-                    )
+                    - energy_cost.aggregate_flow_heat / treat_cost.aggregate_flow_heat
```

Comment:
`energy_cost.aggregate_flow_heat` is negative. That's why it was multiplied by -1.